docker-compose.yml

```diff
@@ -46,6 +46,8 @@ services:
         restart: unless-stopped
         tty: true
         image: ghcr.io/simjanos-dev/linguacafe-python-service:${VERSION:-main}
```

Comment:
Here's an other one too. (I know it's still not ready to merge.)

Reply:
Fixed, good job spotting it.